includes/Core/User_Input/User_Input.php

```diff
@@ -203,6 +203,9 @@ public function are_settings_empty( $settings = array() ) {
 			}
 		}
 
+		// Conversion events can be empty on setup if no event has been detected.
```

Comment:
You are right, it is a bit vague , I expanded on it. But shortly, it is not accounted for settings check which determines if user input was setup or not, since this relates to user input having conversion events or not, and can be present either during setup or added later on.